src/MICore/CommandFactories/MICommandFactory.cs

```diff
@@ -661,6 +670,12 @@ public virtual bool CanDetach()
             return true;
         }
 
+        public virtual async Task<string[]> GetTargetFeatures()
+        {
+            Results results = await _debugger.CmdAsync("-list-target-features", ResultClass.done);
+            return results.Find<ValueListValue>("features").AsStrings;
```

Comment:
The [docs](https://sourceware.org/gdb/current/onlinedocs/gdb/GDB_002fMI-Support-Commands.html#The-_002dlist_002dtarget_002dfeatures-Command) say it's named `result` but in my case gdb returned `features`.